.travis.yml

```diff
@@ -1,13 +1,34 @@
-language: swift
-os: osx
 osx_image: xcode10
-
+language: swift
+cache:
+  directories:
+  - Carthage
+env:
+  global:
+  - FRAMEWORK_NAME=MnemonicKit
+before_install:
+- gem install slather
+- brew update
+- brew outdated carthage || brew upgrade carthage
+- carthage bootstrap --platform iOS --cache-builds
+- pod install || pod install --repo-update
+before_deploy:
+- carthage build --no-skip-current --platform iOS --cache-builds
+- carthage archive $FRAMEWORK_NAME
+- pod trunk push
 script:
-  - xcodebuild -enableCodeCoverage=YES -scheme MnemonicKitTests -workspace MnemonicKit.xcworkspace -destination platform\=iOS\ Simulator,OS\=12.0,name\=iPhone\ X test
-  
+- set -o pipefail && xcodebuild test -scheme MnemonicKit -destination 'platform=iOS Simulator,name=iPhone XS,OS=12.0' ONLY_ACTIVE_ARCH=YES | xcpretty
```

Comment:
I think you want to run the `MnemonicKitTests` scheme. MnemonicKit is only going to build the framework IIUC.

Reply:
I added `MnemonicKitTests` as the framework's test target, so it should be fine, I think.